Source/WebKit/UIProcess/WebAuthentication/fido/CtapNfcDriver.cpp

```diff
@@ -78,6 +80,9 @@ void CtapNfcDriver::transact(Vector<uint8_t>&& data, ResponseCallback&& callback
         respondAsync(WTFMove(callback), WTFMove(apduResponse->data()));
         return;
     }
```

Comment:
This needs to be moved farther up in the function because data got moved on line 61.